common/openapi.yml

```diff
@@ -145,6 +145,25 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+  /api/v1/datasets/{id}/export:
+    post:
+      summary: Export the dataset data into a workspace
+      tags: [ datasets ]
+      parameters:
+        - $ref: '#/components/parameters/Id'
+      operationId: exportDataset
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateExportRequest'
+        required: true
+      responses:
+        '204':
+          description: The data was successfully exported into a workspace
+          content: { }
+        '404':
```

Comment:
Do we need to capture a more specific error?

Reply:
I think we probably want to throw a 403 if a user has 'Discoverer' but not 'Reader' access?

Reply:
Maybe, but TDC shouldn't do any permission check here, it should leave it up to the storage system to do.

Reply:
Agreed that we shouldn't call to any permission check endpoint, but re: what the users of the API should expect, we want to provide possible error cases in our documentation.